### DISCUSSION TITLE
Questions about Labs and QR Control general behaviour [Hero11 Black]

### DISCUSSION THREAD

**Discussion by Fedess1986** [gopro/labs#737] [Q&A] [answered]
I have been reading the Gopro articles, and started using GoPro Labs on my Hero11 Black since a little while, just doing experiments. And I have few questions about the firmware/camera behaviour in differents situations, related to the control through QR code and camera settings.

1. QR Control app has the option for “Not set” but not for OFF in some settings. I don’t know if it is an app limitation or my misunderstanding. For example, if I set “Raw audio” to “not set”, and let’s suppose the camera is already set to “Low”, will the QR code set it to OFF or will it leave that setting as is (“Low”)?

2. If I set LOGB to 400 through QR code, and then I go to camera menu and set colro to “FLAT”, will it override that setting and disable LOGB to set FLAT or how does it behave?

3. If I put a script file into SD card, and after some use I format SD card through camera menu, will the script get deleted? is there some way to temporarily copy the script into any RAM memory of the camera, and after formatting put the file back on SD card?

Thanks in advance =)

**Comment by dnewman-gpsw** [comment, marked as answer]
1. "Not Set" = no command is sent in the QR Code. Leaves the setting in the current state.  For Raw audio use 'aA' or set Wind to 'Auto'.  The Raw audio and Wind are coupled, as this feature changed between camera models, added to the confusing.
2. Anything set with !Mwxyz or oMwxyz is completely independent of the menus, the menu controls will not undo a global hack.  So !MLOGB=400 and switch the color will not have the same effect as !MLOGB=0 (to switch it off.)  !MLOGB=400 and Vibrant will look different than Flat, while this is due to the tone-mapping and saturation, not the curve, which LOGB overrides.  There are interesting "effects" combining global and menu setting.  oMWIDE=1 was designed to be used with Flat or LOGB, yet WIDE Natural is an interesting look. 
3. All on the SD Card will be deleted, including any scripts saved there.  Typically only !MBOOT=... type commands are saved to the SD.

**Reply by Fedess1986** [reply]
thank you!

**Reply by Fedess1986** [reply]
Hi David, would you help me to understand this? It is related to the previous question on this post.

The camera I have is Hero11 Black, and I have been reading [this article](https://gopro.com/en/me/news/hero10-black-image-quality-updates) (from the Hero10 Black time, but I understand it works the same for Hero11).

According to this article mention I understand that, in the camera Menu, if the color is set to:
- Vibrant: LTM is not applied
- Natural: LTM applied
- Flat: I don't know (and would like to know if LTM is applied or not)

Earlier you mentioned:


>  !MLOGB=400 and Vibrant will look different than Flat, while this is due to the tone-mapping and saturation, not the curve, which LOGB overrides. 

Does it mean that Flat uses LTM? 



> There are interesting "effects" combining global and menu setting. oMWIDE=1 was designed to be used with Flat or LOGB, yet WIDE Natural is an interesting look.

In this case you mean color Natural (LOGB=0) and WIDE=1? (if I'm in the correct way, this really goes interesting!)



Thanks in advanced, I'm just trying to fully understand those commands and setting, which confuses me a littile some times

**Reply by dnewman-gpsw** [reply]
Vibrant and Natural both use LTM (and GTM).  Flat uses neither.  Flat is Logbase 113.
